mypy/server/aststripnew.py

```diff
@@ -85,13 +85,16 @@ def prepare_implicit_var_patches(self, node: ClassDef) -> None:
         """Produce callbacks that re-add attributes defined on self."""
         for name, sym in node.info.names.items():
             if isinstance(sym.node, Var) and sym.implicit:
+                explicit_self_type = sym.node.explicit_self_type
+
                 def patch() -> None:
                     existing = node.info.get(name)
+                    current_existing = node.info.names.get(name)
```

Comment:
The name was a bit confusing. "Current" could mean "current class" or "currently". Maybe rename to something like `defined_in_this_class` make this a boolean: `defined_in_this_class = name in node.info.names`. Actually, it might be better to get rid of this variable and move the check below to the if statement.

Reply:
TBH, I would rather keep this defined separately. I don't like large conditions, and with the name the intention is clear.